tiledb/sm/c_api/tiledb.h

```diff
@@ -4562,11 +4570,14 @@ TILEDB_EXPORT int32_t tiledb_array_alloc(
  *
  * @note If the same array object is opened again without being closed,
  *     an error will be thrown.
+ * @note The config should be set before opening an array.
```

Comment:
We should clarify that if one needs to open the array at a specific time interval, they need to set the `timestamp_{start,end}` values at a config and set it to the array object before opening the array.